backend/canisters/storage_bucket/CHANGELOG.md

```diff
@@ -7,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- Add cacheable resource header ([#5947](https://github.com/open-chat-labs/open-chat/pull/5947))
```

Comment:
Typically we put this at the bottom of the list but it doesn't really matter! Hamish might have rejected your PR though ;-)

Reply:
Feel free to resolve this conversation!